solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/LTRQParserPlugin.java

```diff
@@ -44,10 +51,13 @@
  * efi.myCompanyQueryIntent=0.98}
  *
  */
-public class LTRQParserPlugin extends QParserPlugin {
+public class LTRQParserPlugin extends QParserPlugin implements ResourceLoaderAware, ManagedResourceObserver {
   public static final String NAME = "ltr";
 
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  
+  private ManagedFeatureStore fr = null;
+  private ManagedModelStore mr = null;
```

Comment:
LTRQParser has its own ManagedModel/Feature store variables.  Should they be removed and use these new ones here?